### ISSUE TITLE
Show one more screen on dayoff/vacation request submission

### ISSUE THREAD

**Issue by santee** [SoftwareCountry/arcadian-assistant#301]
Just informational screen, stating 'Your vacation (dayoff / sick leave) will be shortly reviewed approved by your supervisor". Text can differ for each type, actually

**Comment by DesmanLead** [comment]
@santee is alert fine?
https://facebook.github.io/react-native/docs/alert

**Comment by antonm76** [comment]
@santee is system dialog box enough?

**Comment by santee** [comment]
better to have it as one more screen in the wizard, don't you think?

**Comment by KuchaLeni** [comment]
Reopened.
Informational screen is not displayed for workout and sick leave requests.

**Comment by sponkrashin** [comment]
> it's strange to say e.g. that your sick leave is to be reviewed and approved

Yes, it is correct flow to exclude accidental events that would be processed by HRs otherwise.

**Comment by DesmanLead** [comment]
@sponkrashin the confirmation dialog is there. The task is about one more additional information dialog with just one "ok" button.

**Comment by sponkrashin** [comment]
I just answered to @antonm76's notice that sick leave shouldn't be approved. I don't know anything about your dialog boxes or other stuff :) But as sick leaves should also be approved, dialog box should be there too, imho.
